### PR TITLE
Tab Spinner - Upload files after running ETL

### PR DESCRIPTION
**Pull request by fbertsch**
Tested, works, and data appears on the dash.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/python_mozetl/29)

### CODE REVIEW

File: mozetl/tab_spinner/generate_counts.py [hunk 1]
```diff
@@ -37,9 +38,12 @@ def run_spinner_etl(sc):
         pings = old_infra_pings.union(new_infra_pings)
         build_results[build_type] = get_short_and_long_spinners(pings)
 
+    s3 = boto3.client('s3')
```
Comment (harterrt) [inline mozetl/tab_spinner/generate_counts.py]: Flake8 is complaining that `s3` is never used and `s3_client` is undefined.

File: mozetl/tab_spinner/generate_counts.py [hunk 2]
```diff
@@ -37,9 +38,12 @@ def run_spinner_etl(sc):
...
     for result_key, results in build_results.iteritems():
-        filename = "./output/severities_by_build_id_%s.json" % result_key
+        filename = "severities_by_build_id_%s.json" % result_key
         results_json = json.dumps(results, ensure_ascii=False)
 
         with open(filename, 'w') as f:
             f.write(results_json)
+
+        s3_client.upload_file(filename, 'telemetry-public-analysis-2', 'spinner-severity-generator/data/{}'.format(filename))
```
Comment (harterrt) [inline mozetl/tab_spinner/generate_counts.py]: Can you shorten this line to 100 char?

Comment (harterrt) [inline mozetl/tab_spinner/generate_counts.py]: This analysis is already kept on a public bucket, correct?

Comment (fbertsch) [inline mozetl/tab_spinner/generate_counts.py]: Yup. Just putting it where it is currently.

**Comment by fbertsch** [comment]
@harterrt apparently I didn't push the changes I made when I ran it on the cluster. Things are now fixed.

**Pull request merged**